_pages/logs_rclog.md

```diff
@@ -39,7 +39,340 @@ jumbotron:
+- 6.11.11-rc1, [allmodconfig builds failed on arm64, arm, riscv and x86_64](https://lore.kernel.org/stable/CA+G9fYtNvEDcUEuv=QFC84y+pXY1UszoRYOitJztCApLV7-psg@mail.gmail.com) (<48h)
+- 6.12.2-rc1, [allmodconfig builds failed on arm64, arm, riscv and x86_64](https://lore.kernel.org/stable/CA+G9fYu21yqTvL428TFueMJ1uU1H_u8Vc470dER2CTrNK=Js0g@mail.gmail.com) (<48h)
+### 2024-11-20
+#### Reported regression (<48h)
+- 6.1.119-rc1,[parisc builds fail](https://lore.kernel.org/stable/CA+G9fYt=ntvdU30W2jZRSUcAqzrKyuCfDH3nDwRB3STd6uOnPw@mail.gmail.com) (<48h)
+#### Reported no regressions (<48h)
+- [6.11.10-rc1](https://lore.kernel.org/stable/CA+G9fYsaZNm2VKnm10tR4dy0DfnE3_FQtkj-E9-SPR-RacXxcw@mail.gmail.com), [6.12.1-rc1](https://lore.kernel.org/stable/CA+G9fYsuQ_F0H8ByKiNazExpVbPGNrZ8amUoXCjc_njwng2Vpg@mail.gmail.com), [6.6.63-rc1](https://lore.kernel.org/stable/CA+G9fYvy3d162b99zNCXug+m=AUGJ7hhoeDNrOFDjHcxqm_XJQ@mail.gmail.com)
+### 2024-11-15
+#### Reported regression (<48h)
+- 5.4.286-rc1, None (SUPERSEEDED)
+#### Reported no regressions (<48h)
+- [4.19.324-rc1](https://lore.kernel.org/stable/CA+G9fYvzExabWp94wW9dT=_KWLUWjTJvT4ZtoiJvFvGyxjW9Gg@mail.gmail.com), [6.11.9-rc1](https://lore.kernel.org/stable/CA+G9fYuzB_AvjT7tZQy+tH257ztf--cs8Y+y0wF25RuruOS+cw@mail.gmail.com), [5.10.230-rc1](https://lore.kernel.org/stable/CA+G9fYtsjLQOQFJsK8BBvEe03B+PJp0dSjpobbT+65Qhgaeoxw@mail.gmail.com), [6.6.62-rc1](https://lore.kernel.org/stable/CA+G9fYuZYe7_AfPgjPMqPL9xgAq90kA5T272B3irJ-hLiM0Acg@mail.gmail.com), [6.1.118-rc1](https://lore.kernel.org/stable/CA+G9fYtKyZtmgoxM+wNws=9WZFc1eMJuM0H0RQoAm+BgKfkSNQ@mail.gmail.com), [5.15.173-rc1](https://lore.kernel.org/stable/CA+G9fYvjgM=J2KCcRPD5xQk6MYMaw4DVPN_-OFhHqBYY-DV-5g@mail.gmail.com), [5.4.286-rc2](https://lore.kernel.org/stable/CA+G9fYtdzDCDP_RxjPKS5wvQH=NsjT+bDRbukFqoX6cN+EHa7Q@mail.gmail.com)
+### 2024-11-12
+#### Reported no regressions (<48h)
+- [6.11.8-rc1](https://lore.kernel.org/stable/CA+G9fYsT27U3v0cDOxRCE53qTDC13f8Tx2QiueD4bWOe-N-nNw@mail.gmail.com), [6.6.61-rc1](https://lore.kernel.org/stable/CA+G9fYuvsU3hfJd_3tDsv1HpB_hBPTpzcyGqJ1bRhUcwmhKMjw@mail.gmail.com), [6.1.117-rc1](https://lore.kernel.org/stable/CA+G9fYvtaB2Zmr6mWr03p2WF-ozf6JVSZVyPj03qEZHt8_VzyQ@mail.gmail.com), [5.15.172-rc1](https://lore.kernel.org/stable/CA+G9fYtM3KGHE8boa1nvhJn-JNTtMob93KGubiXA+f-ak0OSGQ@mail.gmail.com)
+### 2024-11-07
+#### Reported no regressions (<48h)
+- [4.19.323-rc2](https://lore.kernel.org/stable/CA+G9fYs8jLY9t=u+rBJ8e18LbpB10ortb6q8j0r8yRPw6-J=JA@mail.gmail.com), [5.4.285-rc2](https://lore.kernel.org/stable/CA+G9fYuXK1LYzu++g6LUBOEhbOMQSO9Zz6AhM7100NMMEZLuAg@mail.gmail.com), [6.11.7-rc2](https://lore.kernel.org/stable/CA+G9fYtr2V+noAMBzyCafOMOeZf33bo6DKMcGOqsH0-C+0xPEw@mail.gmail.com)
+### 2024-11-06
+#### Reported regression (<48h)
+- 5.4.285-rc1,[arm builds fail](https://lore.kernel.org/stable/CA+G9fYvqfxYX8r1rMcaiaq-K0xuVf4-pRrS77ovNMHNiA+FPkA@mail.gmail.com) (<48h)
```

Comment:
you say "(<48)" here and its already stated in the "#### Reported regression <48h)"...

Reply:
Thanks for the review.
I will update.